ffmpeg/platform/gpl/pom.xml

```diff
@@ -0,0 +1,186 @@
+          <execution>
+            <id>default-jar</id>
+            <configuration>
+              <archive>
+                <manifestEntries>
+                  <Class-Path>${javacpp.moduleId}.jar ${javacpp.moduleId}-linux-x86-gpl.jar ${javacpp.moduleId}-linux-x86_64-gpl.jar ${javacpp.moduleId}-linux-armhf-gpl.jar ${javacpp.moduleId}-linux-arm64-gpl.jar ${javacpp.moduleId}-linux-ppc64le-gpl.jar ${javacpp.moduleId}-macosx-x86_64-gpl.jar ${javacpp.moduleId}-windows-x86-gpl.jar ${javacpp.moduleId}-windows-x86_64-gpl.jar</Class-Path>
+                </manifestEntries>
+              </archive>
+            </configuration>
+          </execution>
+          <execution>
+            <id>empty-javadoc-jar</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <classifier>javadoc</classifier>
+            </configuration>
+          </execution>
+          <execution>
+            <id>empty-sources-jar</id>
+            <goals>
+              <goal>jar</goal>
+            </goals>
+            <configuration>
+              <classifier>sources</classifier>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>add-module-infos</id>
+            <phase>none</phase>
+          </execution>
+          <execution>
+            <id>add-platform-module-info</id>
+            <phase>package</phase>
+            <goals>
+              <goal>add-module-info</goal>
+            </goals>
+            <configuration>
+              <modules>
+                <module>
+                  <file>${project.build.directory}/${project.artifactId}.jar</file>
+                  <moduleInfoSource>
+                    module org.bytedeco.${javacpp.moduleId}.platform {
+//                      requires static org.bytedeco.${javacpp.moduleId}.android.arm;
+//                      requires static org.bytedeco.${javacpp.moduleId}.android.arm64;
+//                      requires static org.bytedeco.${javacpp.moduleId}.android.x86;
+//                      requires static org.bytedeco.${javacpp.moduleId}.android.x86_64;
+                      requires static org.bytedeco.${javacpp.moduleId}.linux.x86;
+                      requires static org.bytedeco.${javacpp.moduleId}.linux.x86_64;
+                      requires static org.bytedeco.${javacpp.moduleId}.linux.armhf;
+                      requires static org.bytedeco.${javacpp.moduleId}.linux.arm64;
+                      requires static org.bytedeco.${javacpp.moduleId}.linux.ppc64le;
+                      requires static org.bytedeco.${javacpp.moduleId}.macosx.x86_64;
+                      requires static org.bytedeco.${javacpp.moduleId}.windows.x86;
+                      requires static org.bytedeco.${javacpp.moduleId}.windows.x86_64;
```

Comment:
The module names here are missing a `.gpl` suffix.

Reply:
Fixed.